django_app/api/serializers.py

```diff
@@ -1,26 +1,14 @@
 from rest_framework import serializers
-from projects.models import BioSample, Observation
+from projects.models import BioSample, Observation, UserCart
```

Comment:
We don't really need serializer for UserCart and we don't have it, that's why it should be deleted